set_2_block_crypto/challenge_11_ecb_cbc_detection_oracle/cat.hpp

```diff
@@ -8,8 +8,7 @@
 
 namespace cryptopals {
 
-class cat_streambuf
-    : public std::streambuf, private boost::noncopyable
+class cat_streambuf : public std::streambuf, private boost::noncopyable
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this inheritance "public". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3469)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant access specifier; it is already "private". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3540)